docs/source/generate.rst

```diff
@@ -41,6 +44,10 @@ Node
         doc.render()
     }
     catch (error) {
+        errorHandler(error);
+    }
+
+    function errorHandler(error) {
         // The error thrown here contains additional information when logged with JSON.stringify (it contains a properties object containing all suberrors).
         function replaceErrors(key, value) {
```

Comment:
Can you put replaceErrors at the top level ?

Reply:
(Same thing for all other places)

Reply:
How do you see the preview of rst file in your editor? Is there an extension for it? I am using Vscode.

Reply:
I personally regenerate the doc locally with : 

```
npm run generate:doc
```

then I run a simple HTTP server with `python -m SimpleHTTPServer 8000` and open the generated file in the browser

Reply:
I tried to do it for some time, could not get it to set up. Currently, I am validating on online Sphinx editor.